routers/router.go

```diff
@@ -0,0 +1,10 @@
+package routers
+
+import (
+	"goMarketer/controllers"
+	"github.com/astaxie/beego"
```

Comment:
Also here, If we decide to follow our Go convention. How about separate this 2 groups by blank line.

Reply:
https://github.com/andyduong1920/go_marketer/pull/1/commits/37ca365970a8d680f8ee429a370a5a800dbcb02e